flang/lib/Optimizer/Transforms/RewriteLoop.cpp

```diff
@@ -22,6 +22,12 @@ static llvm::cl::opt<bool>
                          llvm::cl::desc("disable FIR to CFG pass"),
                          llvm::cl::init(false));
 
+/// minimum trip count is 1, not 0
+static llvm::cl::opt<bool> forceLoopToExecuteOnce(
```

Comment:
Someone might expect this to also apply to unstructured loops.

Reply:
I can write an issue.

Reply:
https://github.com/flang-compiler/f18-llvm-project/issues/214

Reply:
#239